### DISCUSSION TITLE
How can I fetch package installation directory in xmake.lua?

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#1586] [Q&A] [answered]
我想在运行xmake install时从package中copy一些文件，为此需要获取package的安装目录，find_package无法获取由xmake安装的package，有什么其他办法可以获取?

**Comment by waruqi** [comment, marked as answer]
target:pkg("zlib"):installdir()

**Reply by xq114** [reply]
这个接口在文档里面没有，之后可以加上

**Reply by waruqi** [reply]
没啥时间，target实例接口都还没加
